Source/Core/VideoCommon/VertexShaderManager.cpp

```diff
@@ -450,8 +450,11 @@ void VertexShaderManager::SetConstants()
     dirty = true;
   }
 
-  // Handle a potential config change
-  g_freelook_camera.SetControlType(Config::Get(Config::GFX_FREE_LOOK_CONTROL_TYPE));
+  if (g_ActiveConfig.bFreeLook)
```

Comment:
Any reason this can't go here? https://github.com/dolphin-emu/dolphin/blob/master/Source/Core/VideoCommon/RenderBase.cpp#L396

Would save checking every time constants are updated. Not that it'll make much difference.

Reply:
Ah, didn't think of this.  I sort of view that function as just a way to recompile shaders when config changes.  Which is why I did make a similar change in #8832 when freelook specific shaders were added.

I see what you're saying.  This is only temporary though, as #8867 is my plan for freelook in the future.  Still, who knows when that'll get merged, so I'll go ahead and move this for now.